### PR TITLE
Fix readme 's wrong link

### PR DESCRIPTION
**Pull request by rabbit008**
Fix wrong link
Add a repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -345,11 +345,18 @@ Before providing your hacks, you can review the [Contributing Instruction](docs/
...
 
-## **External Repositories**
+## **External Repositories and References**
 Now we have some external usage examples run in NNI from our contributors. Thanks our lovely contributors. And welcome more and more people to join us!
+
+
 * Run [ENAS](examples/tuners/enas_nni/README.md) in NNI
 * Run [Neural Network Architecture Search](examples/trials/nas_cifar10/README.md) in NNI 
-* [Automatic Feature Engineering](examples/trials/auto-feature-engineering/README.md) in NNI
+* [Automatic Feature Engineering](examples/trials/auto-feature-engineering/README.md) in NNI <br>
```
Comment (QuanluZhang) [inline README.md]: How about adding recommenders with this link https://github.com/microsoft/recommenders/blob/master/notebooks/04_model_select_and_optimize/nni_surprise_svd.ipynb ?

File: README.md [hunk 2]
```diff
@@ -345,11 +345,18 @@ Before providing your hacks, you can review the [Contributing Instruction](docs/
...
+
+Besudes, here are some external references (with author's permission) that we would like to share with you.
+
+
+* [Design and implementation of kubeflow/katib](http://gaocegege.com/Blog/机器学习/katib-new)
```
Comment (scarlett2018) [inline README.md]: > Design and implementation of kubeflow/katib
Change to:  Advisor, NNI, Google Vizier 对比总结 (<Design and implementation of kubeflow/katib> 最后章节

File: README.md [hunk 3]
```diff
@@ -345,11 +345,18 @@ Before providing your hacks, you can review the [Contributing Instruction](docs/
 * [Implement customized TrainingService](docs/en_US/TrainingService/HowToImplementTrainingService.md)
 
 
-## **External Repositories**
+## **External Repositories and References**
 Now we have some external usage examples run in NNI from our contributors. Thanks our lovely contributors. And welcome more and more people to join us!
```
Comment (scarlett2018) [inline README.md]: With authors' permission, we listed a set of NNI usage examples and relevant articles.

File: README.md (hunk 1 above)
Comment (rabbit008) [reply README.md]: Ok,fixed , but... do we need permission from the author?

File: README.md [hunk 4]
```diff
@@ -345,11 +345,18 @@ Before providing your hacks, you can review the [Contributing Instruction](docs/
...
+
+
 * Run [ENAS](examples/tuners/enas_nni/README.md) in NNI
```
Comment (scarlett2018) [inline README.md]: * **Repo** - run [ENAS](examples/tuners/enas_nni/README.md) with NNI

File: README.md [hunk 5]
```diff
@@ -345,11 +345,18 @@ Before providing your hacks, you can review the [Contributing Instruction](docs/
...
 * Run [Neural Network Architecture Search](examples/trials/nas_cifar10/README.md) in NNI
```
Comment (scarlett2018) [inline README.md]: * **Repo** run [Neural Network Architecture Search](examples/trials/nas_cifar10/README.md) with NNI

File: README.md (hunk 1 above)
Comment (scarlett2018) [inline README.md]: * **Repo** [Automatic Feature Engineering](examples/trials/auto-feature-engineering/README.md) with NNI <br>

File: README.md [hunk 6]
```diff
@@ -345,11 +345,18 @@ Before providing your hacks, you can review the [Contributing Instruction](docs/
...
+## **External Repositories and References**
 Now we have some external usage examples run in NNI from our contributors. Thanks our lovely contributors. And welcome more and more people to join us!
+
+
 * Run [ENAS](examples/tuners/enas_nni/README.md) in NNI
 * Run [Neural Network Architecture Search](examples/trials/nas_cifar10/README.md) in NNI 
-* [Automatic Feature Engineering](examples/trials/auto-feature-engineering/README.md) in NNI
+* [Automatic Feature Engineering](examples/trials/auto-feature-engineering/README.md) in NNI <br>
+
+Besudes, here are some external references (with author's permission) that we would like to share with you.
```
Comment (scarlett2018) [inline README.md]: remove this line.

File: README.md (hunk 2 above)
Comment (scarlett2018) [reply README.md]: * **Blog** Advisor, NNI, Google Vizier 对比总结: [Design and implementation of kubeflow/katib](http://gaocegege.com/Blog/机器学习/katib-new) 的最后章节

File: README.md (hunk 1 above)
Comment (rabbit008) [reply README.md]: ok fixed , but.. do we need permission from the author ?

File: README.md (hunk 6 above)
Comment (rabbit008) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (rabbit008) [reply README.md]: fixed

File: README.md (hunk 5 above)
Comment (rabbit008) [reply README.md]: fixed

File: README.md (hunk 4 above)
Comment (rabbit008) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (rabbit008) [reply README.md]: fixed

File: README.md [hunk 7]
```diff
@@ -345,11 +345,14 @@ Before providing your hacks, you can review the [Contributing Instruction](docs/
...
-* Run [Neural Network Architecture Search](examples/trials/nas_cifar10/README.md) in NNI 
-* [Automatic Feature Engineering](examples/trials/auto-feature-engineering/README.md) in NNI
+## **External Repositories and References**
+With authors' permission, we listed a set of NNI usage examples and relevant articles.
+
+* **Repo** - Run [ENAS](examples/tuners/enas_nni/README.md) with NNI
+* **Repo** - Run [Neural Network Architecture Search](examples/trials/nas_cifar10/README.md) with NNI 
+* **Repo** - [Automatic Feature Engineering](examples/trials/auto-feature-engineering/README.md) with NNI 
+* **Repo** - [Hyperparameter Tuning for Matrix Factorization](https://github.com/microsoft/recommenders/blob/master/notebooks/04_model_select_and_optimize/nni_surprise_svd.ipynb) with NNI
+* **Blog** - Advisor, NNI, Google Vizier ,[Design and implementation of kubeflow/katib](http://gaocegege.com/Blog/机器学习/katib-new)
```
Comment (scarlett2018) [inline README.md]: **Blog (in Chinese)** - AutoML tools (Advisor, NNI and Google Vizier) comparison by @gaocegege - 总结与分析 section of [Design and implementation of kubeflow/katib](http://gaocegege.com/Blog/机器学习/katib-new)

File: README.md [hunk 8]
```diff
@@ -345,11 +345,14 @@ Before providing your hacks, you can review the [Contributing Instruction](docs/
...
-* Run [Neural Network Architecture Search](examples/trials/nas_cifar10/README.md) in NNI 
-* [Automatic Feature Engineering](examples/trials/auto-feature-engineering/README.md) in NNI
+## **External Repositories and References**
+With authors' permission, we listed a set of NNI usage examples and relevant articles.
+
+* **Repo** - Run [ENAS](examples/tuners/enas_nni/README.md) with NNI
+* **Repo** - Run [Neural Network Architecture Search](examples/trials/nas_cifar10/README.md) with NNI 
+* **Repo** - [Automatic Feature Engineering](examples/trials/auto-feature-engineering/README.md) with NNI 
+* **Repo** - [Hyperparameter Tuning for Matrix Factorization](https://github.com/microsoft/recommenders/blob/master/notebooks/04_model_select_and_optimize/nni_surprise_svd.ipynb) with NNI
+* **Blog (in Chinese)** - [AutoML tools (Advisor, NNI and Google Vizier) comparison by @gaocegege - 总结与分析 section of design and implementation of kubeflow/katib](http://gaocegege.com/Blog/机器学习/katib-new)
```
Comment (gaocegege) [inline README.md]: If we only care about the last section, we can use 
```suggestion
* **Blog (in Chinese)** - [AutoML tools (Advisor, NNI and Google Vizier) comparison by @gaocegege - 总结与分析 section of design and implementation of kubeflow/katib](http://gaocegege.com/Blog/%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0/katib-new#%E6%80%BB%E7%BB%93%E4%B8%8E%E5%88%86%E6%9E%90)
```

Comment (scarlett2018) [reply README.md]: Thanks @gaocegege, that's a better fit. @rabbit008 let's use this section link.

Comment (rabbit008) [reply README.md]: thanks for @gaocegege ,this pr has been merged. I will request a new branch to fix this link.

**Pull request merged**